DataConnectors/SymantecProxySG/Connector_Syslog_SymantecProxySG.json

```diff
@@ -0,0 +1,122 @@
+{ 
+    "id": "SymantecProxySG",
+    "title": "Symantec ProxySG", 
+    "publisher": "Symantec", 
+    "descriptionMarkdown": "The [Symantec ProxySG](https://www.broadcom.com/products/cyber-security/network/gateway/proxy-sg-and-advanced-secure-gateway) allows you to easily connect your Symantec ProxySG logs with Azure Sentinel, to view dashboards, create custom alerts, and improve investigations. Integrating Symantec ProxySG with Azure Sentinel provides more visibility into your organization's network proxy traffic and will enhance security monitoring capabilities.", 
```

Comment:
Add this after descriptionMarkdown to have the banner on the next steps page:

"additionalRequirementBanner": "This data connector dependents on a parser based on Kusto function to work as expected. Follow the steps to use this Kusto functions alias **SymantecProxySG** in queries and workbooks. [Follow steps to get this Kusto function>](https://aka.ms/sentinelgithubparserssymantecproxysg) ",

Reply:
Corrections made and pushed